lib/jekyll/tags/include.rb

```diff
@@ -2,14 +2,6 @@
 
 module Jekyll
   module Tags
-    class IncludeTagError < StandardError
-      attr_accessor :path
-
-      def initialize(msg, path)
-        super(msg)
-        @path = path
-      end
-    end
 
```

Comment:
@ashmaroli a lot of these are in fact Jekyll's own files as some people version their `.bundle` repository 😲